src/OutgoingCallMessage.ts

```diff
@@ -243,9 +222,11 @@ export class OutgoingCallMessage extends EventEmitter implements CallMessage {
       );
     }
 
-    const { message, code } = nativeCallMessageEvent.error;
-    const error = constructTwilioError(message, code);
-    this.emit(OutgoingCallMessage.Event.Failure, error);
+    if (this.getSid() === nativeCallMessageEvent.voiceEventSid) {
```

Comment:
You should do this check on line `208` before you invoke the handler at all. That way you can cover this for every handler without having to redo the check in every handler.